aio/src/styles/_print.scss

```
@@ -112,4 +112,13 @@
     .feature-section img {
         max-width: 70px !important;
     }
+    
+    a::after {
+        content: " (https://angular.io/" attr(href) ") ";
+      }
+      
+      a[href^="http:"]::after,
+      a[href^="https:"]::after {
+        content: " (" attr(href) ") ";
+      }
```

The indentation doesn't look right for this block:

```suggestion
    }

    a[href^="http:"]::after,
    a[href^="https:"]::after {
      content: " (" attr(href) ") ";
    }
```

---

git reset upstream/master does not works for me with the following error 

```
fatal: ambiguous argument 'upstream': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

---

You probably need to set `upstream` as a remote and fetch `master` from `upstream` first.
Something like:

```sh
git remote add upstream https://github.com/angular/angular.git
git fetch upstream master
```

Let's chat "offline".